dizoo/league_demo/league_scheduler.py

```diff
@@ -0,0 +1,172 @@
+
+class Scheduler(object):
```

Comment:
You need to write the corresponding `unittest` for this class using `pytest`, demo is like [this](https://github.com/opendilab/DI-engine/blob/main/ding/rl_utils/tests/test_ppo.py#L26)

Reply:
done.